jsonschema-hyperschema.xml

```diff
@@ -334,6 +334,16 @@
                 </section>
             </section>
 
+            <section title="deprecated">
+                <t>
+                    If it has a value of boolean true, this keyword indicates that the value of the instance is deprecated, and intended for reverse compatability or internal use only, and may become missing in the future.
```

Comment:
"value of the instance is deprecated" is strange phrasing.  Usually you deprecate a field, not a field's value, or you deprecate an entire resource.  So this could apply to any schema with a "self" link (which is basically what determines that the schema represents a resource), or to a field within an object schema.  Or perhaps a position in a tuple schema (when "items" is an array of schemas).

There's something a bit strange there where this is either about a resource or about the context in which the schema appears (the property name or pattern, for instance).